src/test/groovy/org/mockito/release/notes/contributors/ContributorsSerializerTest.groovy

```diff
@@ -0,0 +1,89 @@
+package org.mockito.release.notes.contributors
+
+import spock.lang.Specification
+import spock.lang.Subject
+
+class ContributorsSerializerTest extends Specification {
+
+    @Subject serializer = new ContributorsSerializer(System.getProperty("java.io.tmpdir") + "/contributors.json")
+    def writer = new StringWriter()
+
+    def "serialization and deserialization of one contributor"() {
+        given:
+        def contributors = new DefaultContributorsSet()
+        def contributor = new DefaultContributor("myName", "myLogin", "myProfileUrl")
+        contributors.addContributor(contributor)
+
+        when:
+        serializer.serialize(contributors)
+        def actual = serializer.desrialize()
+
+        then:
+        actual.getAllContributors().containsAll(contributors.getAllContributors())
+        contributors.getAllContributors().containsAll(actual.getAllContributors())
+    }
+
+    def "serialization and deserialization of zero contributors"() {
+        given:
+        def contributors = new DefaultContributorsSet()
+
+        when:
+        serializer.serialize(contributors)
+        def actual = serializer.desrialize()
+
+        then:
+        actual.getAllContributors().containsAll(contributors.getAllContributors())
+        contributors.getAllContributors().containsAll(actual.getAllContributors())
+    }
+
+    def "serialization and deserialization of 2 contributors"() {
+        given:
+        def contributors = new DefaultContributorsSet()
+        for (int i = 0; i < 2; i++) {
```

Comment:
Let's try to avoid loops and conditionals in tests, even at the cost of tiny bit of duplication. DAMP vs DRY in tests :) Let's optimize for readability when writing tests.